### PR TITLE
APIGOV-28365 - documentation for added discovery mode EventHub

### PR DESCRIPTION
**Pull request by sbolosan**
Add necessary documentation for new discovery mode for Azure ground agent

http://pr-492.amplify-central.opendocs-builder.pcloud.axway.int/

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | 54cae2d |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-492.amplify-central.opendocs-builder.pcloud.axway.int |

File: content/en/docs/connect_manage_environ/connect_azure_gateway/_index.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ The Traceability Agent sends log information about APIs that have been discovere
 * An Axway Amplify subscription in the Amplify platform
 * A Platform Service Account. See [Managing service accounts](https://docs.axway.com/bundle/platform-management/page/docs/management_guide/organizations/managing_organizations/index.html#managing-service-accounts)
 * An Amplify environment. See [Create an environment](/docs/integrate_with_central/cli_central/cli_environments/)
-* An Azure Service principal for the Discovery Agent to use Azure APIs
+* An Azure Service principal for the Discovery Agent to use Azure APIs in default mode or an Event Hub for when the agent runs in EventHub mode
```
Comment (robertengaxway) [inline content/en/docs/connect_manage_environ/connect_azure_gateway/_index.md:44]: Does this read better?
An Azure Service principal for the Discovery Agent to use Azure APIs in default mode or to use an Event Hub for when the agent runs in EventHub mode

File: content/en/docs/connect_manage_environ/connect_azure_gateway/_index.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ These two agents will help you to represent and expose your API Management eco-s
 
 ### Discovery Agent
 
-The Discovery Agent is used to discover new published APIs. The Discovery Agent pushes both REST and SOAP API definitions to Amplify.
+The Discovery Agent is used to discover new published APIs. The Discovery Agent functions in two modes, default mode where all APIs are discovered from the API Manager, and the EventHub mode where the APIs are discovered from the Event Hub Schema Registry. After the discovery process, the agent pushes both REST and SOAP API definitions to Amplify.
```
Comment (robertengaxway) [inline content/en/docs/connect_manage_environ/connect_azure_gateway/_index.md:27]: are we discovering Topics/APIs in EventHub mode?

File: content/en/docs/connect_manage_environ/connect_azure_gateway/_index.md (hunk 1 above)
Comment (sbolosan) [reply content/en/docs/connect_manage_environ/connect_azure_gateway/_index.md:44]: sounds good to me... updated

File: content/en/docs/connect_manage_environ/connect_azure_gateway/_index.md (hunk 2 above)
Comment (sbolosan) [reply content/en/docs/connect_manage_environ/connect_azure_gateway/_index.md:27]: updated

**Review by robertengaxway — APPROVED** [review]
Approved

**Pull request merged**